### ISSUE TITLE
[Question]: bert MLM预训练问题

### ISSUE THREAD

**Issue by xiaotingyun** [PaddlePaddle/PaddleNLP#4997]
### 请提出你的问题

bert在进行MLM任务时，是否需要将mask的token处的attention_mask设为0。例如text = ['我','[MASK]','你']，attention_mask是否应该为[1,0,1]

**Comment by xiaotingyun** [comment]
在GPT预训练过程中找到了答案，不需要将attention_mask设为0，Bert通过[MASK]来代表遮盖，而GPT通过设置attention_mask为0在计算自注意力时自动屏蔽后面的token，这是两种为了实现遮盖不一样的实现方法。

**Comment by tiesanguaixia** [comment]
> 在GPT预训练过程中找到了答案，不需要将attention_mask设为0，Bert通过[MASK]来代表遮盖，而GPT通过设置attention_mask为0在计算自注意力时自动屏蔽后面的token，这是两种为了实现遮盖不一样的实现方法。

您好！可以请教下ERNIE 1.0的phrase&named entity知识增强masking策略有开源实现的代码吗？是哪个文件呀

**Comment by xiaotingyun** [comment]
> > 在GPT预训练过程中找到了答案，不需要将attention_mask设为0，Bert通过[MASK]来代表遮盖，而GPT通过设置attention_mask为0在计算自注意力时自动屏蔽后面的token，这是两种为了实现遮盖不一样的实现方法。
> 
> 您好！可以请教下ERNIE 1.0的phrase&named entity知识增强masking策略有开源实现的代码吗？是哪个文件呀

https://github.com/PaddlePaddle/PaddleNLP/tree/develop/model_zoo/ernie-1.0
